src/controllers/userControllers.ts

```diff
@@ -53,16 +62,17 @@ class UserController {
           message: "User created Successfully",
           data: _.pick(user, ["_id", "name", "email"]),
         });
-    } catch (error) {
-      console.log(error + " You are catching this");
+    } catch (error: any) {
+      console.log(error + " You are catching this in userController");
+      //let err = createError("You  are in  User Controller", 400);
       return res.status(error.code).json(error.message);
```

Comment:
following the pattern i discussed before

Reply:
And be consistent every where, or you can create a function for response and resuse it everywhere

const Response = ({ res, code, message, data, token }) => res.status(code).json({ message, data, token })